addons/circulation/functions/fnc_fullHealLocal.sqf

```diff
@@ -22,6 +22,9 @@ _patient setVariable [QGVAR(cprCount), 2, true];
 _patient setVariable [QGVAR(heartRestart), false, true];
 _patient setVariable [QGVAR(cardiacArrestType), 0, true];
 
+_patient setVariable [QGVAR(deviceCode), 0, true];
```

Comment:
why do we need to clear this on fullHeal? (question to both variables, i do quick review and cant check this one big function rn)